### PR TITLE
Prepare qualified identifiers

### PR DESCRIPTION
**Pull request by privat**
First step to solve #1260: do not lose qualifiers in the AST.

This is just a API change without new features. qualifiers are still ignored but now they are in the AST.

### CODE REVIEW

File: src/parser/parser_nodes.nit [hunk 1]
```diff
@@ -1648,6 +1648,16 @@ class ABraassignMethid
 	var n_assign: TAssign is writable, noinit
 end
 
+# A qualified simple identifier
```
Comment (Morriar) [inline src/parser/parser_nodes.nit]: You should add an example like for `ASignature`

**Comment by Morriar** [comment]
+1

I'm surprised that there is no more use og meth.n_id than this.

**Comment by privat** [comment]
reroll: added suggested example

**Comment by R4PaSs** [comment]
:shipit:

**Pull request merged**